pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -3,999 +3,999 @@ source: pdg/src/main.rs
-	n[12]: field.0    n[3]  => _12 @ bb4[18]: fn simple;
-	n[13]: addr.load  n[12] => _   @ bb4[18]: fn simple;
-	n[14]: field.1    n[3]  => _13 @ bb4[21]: fn simple;
-	n[15]: addr.load  n[14] => _   @ bb4[21]: fn simple;
-	n[16]: addr.load  n[3]  => _   @ bb4[24]: fn simple;
-	n[17]: field.2    n[3]  => _   @ bb4[25]: fn simple;
-	n[18]: addr.store n[17] => _   @ bb4[25]: fn simple;
-	n[19]: copy       n[3]  => _16 @ bb4[30]: fn simple;
-	n[20]: copy       n[19] => _2  @ bb0[0]:  fn recur; 
-	n[21]: copy       n[20] => _13 @ bb8[4]:  fn recur; 
-	n[22]: copy       n[21] => _2  @ bb0[0]:  fn recur; 
-	n[23]: copy       n[22] => _13 @ bb8[4]:  fn recur; 
-	n[24]: copy       n[23] => _2  @ bb0[0]:  fn recur; 
-	n[25]: copy       n[24] => _13 @ bb8[4]:  fn recur; 
-	n[26]: copy       n[25] => _2  @ bb0[0]:  fn recur; 
-	n[27]: copy       n[26] => _8  @ bb1[3]:  fn recur; 
-	n[28]: copy       n[27] => _7  @ bb1[4]:  fn recur; 
-	n[29]: free       n[28] => _0  @ bb1[5]:  fn recur; 
-	n[30]: copy       n[26] => _14 @ bb9[5]:  fn recur; 
-	n[31]: copy       n[26] => _14 @ bb9[5]:  fn recur; 
-	n[32]: copy       n[26] => _14 @ bb9[5]:  fn recur; 
-	n[33]: addr.load  n[1]  => _   @ bb5[3]:  fn simple;
-	n[34]: addr.store n[3]  => _   @ bb5[7]:  fn simple;
+	n[0]:  alloc      _     => _6  @ bb3[2]:  fn simple; _6 = pointers::malloc(move _7)             
+	n[1]:  copy       n[0]  => _5  @ bb4[2]:  fn simple; _5 = move _6 as *mut pointers::S (Misc)    
+	n[2]:  copy       n[1]  => _10 @ bb4[9]:  fn simple; _10 = _5                                   
+	n[3]:  copy       n[2]  => _1  @ bb4[10]: fn simple; _1 = move _10                              
+	n[4]:  field.0    n[3]  => _   @ bb4[11]: fn simple; ((*_1).0: i32) = const 10_i32              
+	n[5]:  addr.store n[4]  => _   @ bb4[11]: fn simple; ((*_1).0: i32) = const 10_i32              
+	n[6]:  field.0    n[3]  => _11 @ bb4[13]: fn simple; _11 = ((*_1).0: i32)                       
+	n[7]:  addr.load  n[6]  => _   @ bb4[13]: fn simple; _11 = ((*_1).0: i32)                       
+	n[8]:  field.0    n[1]  => _   @ bb4[14]: fn simple; ((*_5).0: i32) = move _11                  
+	n[9]:  addr.store n[8]  => _   @ bb4[14]: fn simple; ((*_5).0: i32) = move _11                  
+	n[10]: field.1    n[3]  => _   @ bb4[16]: fn simple; ((*_1).1: u64) = const 9_u64               
+	n[11]: addr.store n[10] => _   @ bb4[16]: fn simple; ((*_1).1: u64) = const 9_u64               
+	n[12]: field.0    n[3]  => _12 @ bb4[18]: fn simple; _12 = ((*_1).0: i32)                       
+	n[13]: addr.load  n[12] => _   @ bb4[18]: fn simple; _12 = ((*_1).0: i32)                       
+	n[14]: field.1    n[3]  => _13 @ bb4[21]: fn simple; _13 = &raw const ((*_1).1: u64)            
+	n[15]: addr.load  n[14] => _   @ bb4[21]: fn simple; _13 = &raw const ((*_1).1: u64)            
+	n[16]: addr.load  n[3]  => _   @ bb4[24]: fn simple; _14 = &raw const (*_1)                     
```

Comment:
Assuming the new statement listing is accurate, these two `LoadAddr`s should not be here, since `x = &(*y).1` does not access memory.

Reply:
addressed in #565